src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/combination.yml

```diff
@@ -39,13 +39,13 @@ admin_products_combinations_edit_combination:
     requirements:
         combinationId: \d+
 
-admin_products_combinations_remove_combination:
-    path: /combinations/{combinationId}/remove
+admin_products_combinations_delete_combination:
```

Comment:
It's a BC Break as this route has been defined in 178x. Could you document it ?

Reply:
:+1: updated in the PR description